Sources/python/SimEx/Calculators/TNSAIonMatterInteractor.py

```diff
@@ -0,0 +1,163 @@
+from SimEx.Calculators.AbstractIonInteractor import AbstractIonInteractor
+import sdf
```

Comment:
The travis error is: 

> ModuleNotFoundError: No module named 'sdf'

which is obviously related to this line. There is no `sdf` module found in the Travis environment. @leczzsolt is `sdf` a package which can be installed by `conda` or `pip`? If so, please add it into the [requirements.txt](https://github.com/PaNOSC-ViNYL/SimEx/blob/master/requirements.txt)

Reply:
The sdf package is added to the repository, but it is not in conda or pip (as far as I know).

Reply:
To solve the problem, the environment for `sdf` should be set for the Travis environment, @CFGrote do you know how to do it?

Hi @leczzsolt, I just noticed that you merged the branch without passing the test. The problem doesn't seem really solved. @CFGrote , I think we should pass the test before merging? Could you revert the merging?